### PR TITLE
:loud_sound: N°5043 Improve logs

### PR DESCRIPTION
**Pull request by piRGoif**
Improve logs by : 
* adding a log header in MailInboxesEmailProcessor::ListEmailSources : the logs generated in MailInboxBase::GetEmailSource for each mailbox was misleading, you could have the impression that the extension was first connecting to each mailboxes one by one !
* adding a log in EmailBackgroundProcess so that we know when the process exists because time limit is exceeded

### CODE REVIEW

**Comment by piRGoif** [comment]
rebased, solve conflicts and force pushed

File: classes/mailinboxesemailprocessor.class.inc.php [hunk 1]
```diff
@@ -68,17 +68,17 @@ public static function Trace($sText)
...
-	{		
+	{
+		static::Trace("-----------------------------------------------------------------------------------------");
+		static::Trace("Listing sources:");
+
 		$aSources = array();
 		$oSearch = new DBObjectSearch('MailInboxBase');
 		$oSearch->AddCondition('active', 'yes');
 		$oSet = new DBObjectSet($oSearch);
-		/** @var \MailInboxBase $oInbox */
```
Comment (Hipska) [inline classes/mailinboxesemailprocessor.class.inc.php:76]: This comment is useful for IDE autocomplete.

Comment (piRGoif) [reply classes/mailinboxesemailprocessor.class.inc.php:76]: Woops, good catch thanks !
Fixed in 5cbd009f

**Comment by Molkobain** [comment]
Accepted in technical review

File: classes/emailbackgroundprocess.class.inc.php [hunk 2]
```diff
@@ -37,9 +37,13 @@ class EmailBackgroundProcess implements iBackgroundProcess
 	 * @var EmailSource
 	 */
 	private $oCurrentSource;
-	
+
+	/** @var string Message to print before exiting, when the iProcess time limit is exceeded */
```
Comment (Molkobain) [inline classes/emailbackgroundprocess.class.inc.php:41]: Maybe add the `@since` ?

Comment (piRGoif) [reply classes/emailbackgroundprocess.class.inc.php]: Right !
Done in 7ecd7595

**Comment by Molkobain** [comment]
Accepted in functional review

**Pull request merged**